### PR TITLE
// Fix missing $.growl and new UI integration

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
splatoon_009

### CODE REVIEW

File: admin-dev/themes/default/css/bundle/module/module.css [hunk 1]
```diff
@@ -5,6 +5,10 @@
         height:50px;
 }
 
+.btn.onclic {
```
Comment (djfm) [inline admin-dev/themes/default/css/bundle/module/module.css]: onclick?

Comment (Quetzacoalt91) [inline admin-dev/themes/default/css/bundle/module/module.css]: nope, this is really `.onclic` in the UI-kit 

https://github.com/PrestaShop/prestashop-ui-kit/blob/4e7cb7e56c0644b94a0f4af9f11c0ae907e66301/scss/modules/_spinner.scss#L132-L163

Comment (xBorderie) [inline admin-dev/themes/default/css/bundle/module/module.css]: So it's a UI Kit issue. @gfanton? @Mekki78? Anyone? @Bueller?

Comment (Shudrum) [inline admin-dev/themes/default/css/bundle/module/module.css]: Yep, should be fixed before merging.

Comment (Quetzacoalt91) [inline admin-dev/themes/default/css/bundle/module/module.css]: Rename `onclic` > `onclick`

**Pull request merged**